daemon/systemd/sailjaild.service

```diff
@@ -0,0 +1,12 @@
+[Unit]
+Description=Application Permission Management Daemon
+Requires=dbus.service
+After=dbus.socket
+RequiresMountsFor=/home
+[Service]
```

Comment:
nit: I think this should have a space before new section

Reply:
added separator line